assets/js/slider.js

```diff
@@ -0,0 +1,53 @@
+import $ from 'jquery';
+
+export function slider () {
+  $( document ).ready(function() {
+    // Check if jQuery ui is enable
+    if ($.ui.slider) {
```

Comment:
This check won't work as expected, if jQuery UI is not imported in the page, this will throw an error.